src/legacy/core_plugins/kibana/public/management/sections/settings/components/field/field.test.tsx

```diff
@@ -414,32 +431,35 @@ describe('Field', () => {
     } else {
       describe(`for changing ${type} setting`, () => {
         const { wrapper, component } = setup();
+        // @ts-ignore
         const userValue = userValues[type];
         const fieldUserValue = type === 'array' ? userValue.join(', ') : userValue;
 
         if (setting.validation) {
+          // @ts-ignore
           const invalidUserValue = invalidUserValues[type];
           it('should display an error when validation fails', async () => {
-            component.instance().onFieldChange({ target: { value: invalidUserValue } });
+            (component.instance() as Field).onFieldChange({ target: { value: invalidUserValue } });
```

Comment:
As `this.onFieldChange` method is only the 'helper' method for the methods `onFieldChangeSwitch` and `onFieldChangeEvent` (these ones are attached to listeners) we might want to replace this one to the above? (they are used 4 times in this file)

Also, that would fix broken tests :)

Reply:
Very helpful, fixed!